CHANGELOG.md

```diff
@@ -2,7 +2,8 @@
 
 Select a Node.js version below to view the changelog history:
 
-* [Node.js 11](doc/changelogs/CHANGELOG_V11.md) - **Current**
+* [Node.js 12](doc/changelogs/CHANGELOG_V12.md) - **Current**
+* [Node.js 11](doc/changelogs/CHANGELOG_V11.md) - Current
```

Comment:
11 should be `End-of-Life` I think.

Reply:
Not yet

Reply:
… but I think `Maintenance` would be accurate?

Reply:
Yes, I think it should. See the graphic on https://nodejs.org/en/about/releases/ but in the table, version 11 is with "current" label.

Reply:
Ok, then keep it. It's a bit weird we have two "Current" versions, but I guess that's intended.